### ISSUE TITLE
Missing 'Repositories' link

### ISSUE THREAD

**Issue by yogeek** [boyter/searchcode-server#203]
The link `Repositories` that is present in the demo page and pointing to http://demo.searchcodeserver.com/repository/list/ is missing in my instance of searchcode community launched in a docker image. And when I enter the URL directly, I have a 404 error on the page.
Can you explain me how to configure it please ?

**Comment by boyter** [comment]
Ah that would be because I have not updated that docker image in quite a while. When I have a decent internet connection ill have to fix that.

The repository page was added in version 1.3.8 and I suspect the community image is older then that.

**Comment by yogeek** [comment]
@boyter thanks for your answer. Would be nice to enable the autobuild feature of  your dockerhub repository to automatically build your docker image as soon as your code is updated. With it, your docker image would be up-to-date effortlessly :-)

**Comment by boyter** [comment]
Yes, another thing I need to do :) If you can supply some documentation for it that would help. I won't be able to get onto this for the next week sorry unless I can find something quickly :)

**Comment by yogeek** [comment]
Ok no problem ! Here is the doc to set up AutoBuild in Dockerhub : https://docs.docker.com/docker-hub/builds/

**Comment by boyter** [comment]
Neat. Ill look into that when I get back from a break.
